### DISCUSSION TITLE
How to configure DPI setting (for HiRes/4k displays)?

### DISCUSSION THREAD

**Discussion by jo-so** [pinnacle-comp/pinnacle#184] [Q&A]
Using Pinnacle with a HiRes display (3840×2160 pixels on 1016×572 millimetres) gives a very small font size and everything is too small. On Gnome, I have to set a scaling factor of 3. How can I set such a scaling factor for Pinnacle?

**Comment by Ottatop** [comment]
You'll have to wait until #182 gets merged to set a scaling factor. Should be fairly soon.
